### PR TITLE
Add test coverage

### PR DESCRIPTION
**Pull request by adam-thomas**
Adding tests to `django-admin-sso` is better when we have a coverage report to track progress.  Let's do that.

### CODE REVIEW

**Comment by adam-thomas** [comment]
@incuna/backend review/merge please?

File: setup.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
         'Django>=1.4',
         'python-openid>=2.2.5',
         'django-discover-runner>=1.0',
+        'coverage==3.7.1',
     ),
```
Comment (meshy) [inline setup.py:35]: Test requirements should not go in the `install_requires`. If they do, they'll get installed alongside every installation of this, even for people with no intention of running the tests. Instead, they should probably go in the `.travis.yml` (or `test_requirements.txt`) depending on what the package owner prefers.

Comment (adam-thomas) [inline setup.py:35]: @meshy ok fair, cheers :) How's it done in travis.yml?

Comment (adam-thomas) [inline setup.py:35]: Ignore that, incuna-auth to the rescue :P

Comment (adam-thomas) [inline setup.py:35]: Fixed in master.

**Pull request merged**